test/models/podcast_import_test.rb

```diff
@@ -90,6 +91,31 @@
     importer.import
   end
 
+  describe 'episodes only' do
+
+    before {
+      importer.config[:episodes_only] = true
+    }
+
+    it 'must have series set' do
+      exception = -> { importer.import }.must_raise(RuntimeError)
+      exception.message.must_be :start_with?, 'No series'
+    end
+
+    it 'must have podcast distribution' do
+      series.distributions.delete_all
+      importer.series = series
+      exception = -> { importer.import }.must_raise(RuntimeError)
+      exception.message.must_be :start_with?, 'No podcast distribution'
+    end
+
+    it 'imports with a series and podcast' do
+      # series.distributions << create(:podcast_distribution, distributable: series)
```

Comment:
do you need this commented out line?

Reply:
nope, needed it before I broke out this test, will `rm`